controller/iteration_blackbox_test.go

```diff
@@ -736,7 +726,7 @@ func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space
 			SpaceID:    spaceObj.ID,
 			StartAt:    &start,
 			EndAt:      &end,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c